### PR TITLE
Analytics UI fixes

### PR DESCRIPTION
**Pull request by Saranya-jena**
## Proposed changes
- Added analytics UI fixes 
Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx [hunk 1]
```diff
@@ -37,24 +37,20 @@ const TableData: React.FC<TableDataProps> = ({ data }) => {
...
+            context: {data.context ? data.context : 'N/A'}
           </Typography>
         ) : (
           <></>
         )}
       </StyledTableCell>
       <StyledTableCell>
         <Typography variant="body2">
-          <strong>{data.exp_name}</strong>
+          <strong>{data.exp_name ? data.exp_name : 'N/A'}</strong>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:48]: data.exp_name ?? 'N/A'

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx [hunk 2]
```diff
@@ -37,24 +37,20 @@ const TableData: React.FC<TableDataProps> = ({ data }) => {
         </Typography>
         {data?.context !== '' ? (
           <Typography variant="body2" className={classes.context}>
-            context: {data.context}
+            context: {data.context ? data.context : 'N/A'}
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:40]: data.context ?? 'N/A'

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx [hunk 3]
```diff
@@ -37,24 +37,20 @@ const TableData: React.FC<TableDataProps> = ({ data }) => {
...
         </Typography>
       </StyledTableCell>
       <StyledTableCell>
         <ExperimentStatus
-          status={
-            data.test_result !== 'Awaited' && data.test_result !== 'N/A'
-              ? `${data.test_result}ed`
-              : data.test_result
-          }
+          status={data.test_result ? data.test_result : 'N/A'}
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:53]: data.test_result ?? 'N/A'

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx [hunk 4]
```diff
@@ -173,7 +175,7 @@ const TableToolBar: React.FC<TableToolBarProps> = ({
         variant="outlined"
         className={`${classes.formControl} ${classes.testForm}`}
       >
-        <InputLabel className={classes.selectText}> Test Name </InputLabel>
+        <InputLabel className={classes.selectText}>Experiment Name</InputLabel>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx:178]: Please add translations before GA

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx [hunk 5]
```diff
@@ -127,7 +127,9 @@ const TableToolBar: React.FC<TableToolBarProps> = ({
         variant="outlined"
         className={`${classes.formControl} ${classes.testResultForm}`}
       >
-        <InputLabel className={classes.selectText}> Test Result </InputLabel>
+        <InputLabel className={classes.selectText}>
+          Experiment verdict
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx:131]: Translations

File: litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowComparisonTable/styles.ts [hunk 6]
```diff
@@ -295,11 +293,9 @@ const useStyles = makeStyles((theme) => ({
   },
 
   export: {
-    verticalAlign: 'middle',
-    display: 'inline-flex',
-    textAlign: 'center',
+    alignItems: 'center',
+    display: 'flex',
     borderRadius: 3,
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowComparisonTable/styles.ts:301]: Unit ?

File: litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/index.tsx [hunk 7]
```diff
@@ -72,6 +76,20 @@ const ScheduleAndRunStats: React.FC<ScheduleAndRunStatsProps> = ({
...
+        <InputLabel className={classes.selectText} />
+        <Select
+          value={filters}
+          onChange={(event) => {
+            setFilters(event.target.value as Filter);
+          }}
+          className={classes.selectText}
+        >
+          <MenuItem value={Filter.Monthly}>Monthly</MenuItem>
+          <MenuItem value={Filter.Daily}>Daily</MenuItem>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/index.tsx:89]: Translations ?

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]
@Saranya-jena PTAL if some issues can be resolved before merging.

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx (hunk 1 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:48]: Nullish coalasing doesn't check empty strings...

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx (hunk 2 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:40]: Same

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx (hunk 3 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:53]: Same

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableData.tsx:48]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx (hunk 4 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx:178]: translations will go as a separate PR, this is just for UI bug fixes

File: litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/index.tsx (hunk 7 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/ScheduleAndRunStats/index.tsx:89]: same

File: litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx (hunk 5 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx:131]: same

**Pull request merged**